docs/get-started/sumo-logic-ui-new.md

```diff
@@ -40,21 +40,29 @@ This page provides an overview of Sumo Logic **New UI**, currently in beta (slat
         allowfullscreen
         />
 
+
+## Benefits
+
+With the New UI, you can expect:
+
+* Unified experience across Operational and Security analytics products
+* Accelerated performance, reduced load time, and enhanced caching
+* Stateful URLs to preserve page states upon re-login
+* Use-case-based navigation for streamlined feature discovery
```

Comment:
```suggestion
* Unified experience across Operational and Security analytics products.
* Accelerated performance, reduced load time, and enhanced caching.
* Stateful URLs to preserve page states upon re-login.
* Use-case-based navigation for streamlined feature discovery.
```

Reply:
I added periods to follow the recommended style for bulleted items that complete a sentence (for example, see [Microsoft's guidelines](https://www.microsoft.com/en-us/microsoft-365-life-hacks/writing/punctuating-bullet-point-lists#:~:text=You'll%20want%20to%20use,any%20other%20sentence%20you%20write.)).

Reply:
Thank you, @jpipkin1 !